docs/telemetry.md

```diff
@@ -18,6 +18,7 @@ As part of the event ping the most recent state of the user's setting is sent (d
 | Turbo mode currently enabled?      | tracking_protection_enabled     | **true**/false
 | Total home tile count    | total_home_tile_count           | `<int>`
 | Custom home tile count   | custom_home_tile_count          | `<int>`
+| App ID                   | app_id                          | `<string>`
```

Comment:
Nit: can you provide an example of this, or clarify that this is a public, published app-id so it's clear that isn't tied to the user?